composer.json

```diff
@@ -40,7 +40,7 @@
         "symfony/twig-bridge": "^2.3.5 || ^3.0",
         "symfony/validator": "^2.3 || ^3.0",
         "twig/extensions": "^1.0",
-        "twig/twig": "^1.26"
+        "twig/twig": "^1.28"
```

Comment:
I think the rationale behind that was breaking the promise that the Sonata 3.x release would be compatible with Symfony 2.3 until the end, for instance, [which might not happen if we want Twig 2 compatibility](https://github.com/sonata-project/SonataCoreBundle/pull/371). Not a real BC-break since Composer wouldn't let people upgrade Sonata in that case, some people would argue, but in the end, I think it was decided we wouldn't let that happen for big deps like sf or php.